### DISCUSSION TITLE
Does the virtual list itself come with animations?

### DISCUSSION THREAD

**Discussion by ayangweb** [TanStack/virtual#773] [Q&A]
Hey👋, excuse me, is the list animation self contained? Why I don't see a config item in the documentation to turn it on or off, I'm curious to know where this animation comes from?

Code:

![code](https://github.com/user-attachments/assets/7acf7d3f-bc93-48e5-8587-a00af79ec6a4)

Video Effect:

https://github.com/user-attachments/assets/2ece54be-e68e-4256-9655-f49035623fcc

**Comment by piecyk** [comment]
> Hey👋, excuse me, is the list animation self contained? Why I don't see a config item in the documentation to turn it on or off, I'm curious to know where this animation comes from?

No, virtualizer is not adding any styles, they are controller outside.

**Reply by ayangweb** [reply]
> > Hey👋, excuse me, is the list animation self contained? Why I don't see a config item in the documentation to turn it on or off, I'm curious to know where this animation comes from?
> 
> No, virtualizer is not adding any styles, they are controller outside.

After my troubleshooting, it turns out that the animation is after adding the `transition` property to the `Item` component, thanks!
